.devcontainer/Dockerfile

```diff
@@ -7,6 +7,8 @@
 ARG VARIANT="jammy"
 FROM mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
 
+ARG TARGETARCH
```

Comment:
If the user doesn't provide a value, how the build would respond?

Reply:
This is set automatically by buildkit. I've added a comment to point this out